components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Testing/src/Servers/Common/MonitoredNode.cs

```diff
@@ -27,49 +27,58 @@
  * http://opcfoundation.org/License/MIT/1.00/
  * ======================================================================*/
 
-namespace Opc.Ua.Sample {
-    using System.Collections.Generic;
-    using Opc.Ua.Server;
+using System;
+using System.Collections.Generic;
+using Opc.Ua.Server;
 
+namespace Opc.Ua.Sample {
     /// <summary>
     /// Keeps track of the monitored items for a single node.
     /// </summary>
     public class MonitoredNode {
-
+        #region Constructors
         /// <summary>
         /// Initializes the instance with the context for the node being monitored.
         /// </summary>
         public MonitoredNode(
             IServerInternal server,
             INodeManager nodeManager,
             NodeState node) {
-            Server = server;
-            NodeManager = nodeManager;
-            Node = node;
+            m_server = server;
+            m_nodeManager = nodeManager;
+            m_node = node;
         }
+        #endregion
 
+        #region Public Properties
         /// <summary>
         /// The server that the node belongs to.
         /// </summary>
-        public IServerInternal Server { get; }
+        public IServerInternal Server {
+            get { return m_server; }
+        }
 
         /// <summary>
         /// The node manager that the node belongs to.
         /// </summary>
-        public INodeManager NodeManager { get; }
+        public INodeManager NodeManager {
+            get { return m_nodeManager; }
+        }
 
         /// <summary>
         /// The node being monitored.
         /// </summary>
-        public NodeState Node { get; }
+        public NodeState Node {
+            get { return m_node; }
+        }
 
         /// <summary>
         /// Whether the node has any active monitored items for the specified attribute.
         /// </summary>
         public bool IsMonitoringRequired(uint attributeId) {
-            if (_monitoredItems != null) {
-                for (var ii = 0; ii < _monitoredItems.Count; ii++) {
-                    var monitoredItem = _monitoredItems[ii];
+            if (m_monitoredItems != null) {
```

Comment:
coding convention

Reply:
This file is completely taken-over from the OPC UA Stack. Applying the coding conventions of the IIoT project will make a future update very difficult.